agent/Dockerfile

```diff
@@ -1,5 +1,4 @@
-# Install nsenter move to ubuntu:14.04.1
-FROM ibuildthecloud/ubuntu-core-base:14.04-1
+FROM ubuntu:14.04.1
```

Comment:
Allright then; I also mentioned it because some people were under the impression that `14.04` would be *pinned* to `14.04.0` whereas it is basically `14.04.*`

Doing it for reproducibility is a good reason to pin it